### ISSUE TITLE
Network Connection Detected Icon

### ISSUE THREAD

**Issue by citizenanalog** [BitcoinQnA/seedtool#44]
Running v2.0.1 with an offline version of Tails, the Network Connection Detected icon is present. Is this expected behavior?

**Comment by SuperPhatArrow** [comment]
Yes unfortunately. Tor browser does not provide the network API. I am sure this is for a good privacy focused reason. We could test for a live network connection by pinging a server but we decided that users would not like us to make calls to a server as it could be seen as some kind of tracking or data leak.

If there is a way to get the network indicator to work on tor browser, without looking or feeling dodgy, we would like to fix it.

Thank you for the report. I will leave this issue open in the hope that someone can figure out a way to get it working.

**Comment by citizenanalog** [comment]
Great explanation, thanks!
